jkube-kit/resource/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmConfigTest.java

```diff
@@ -140,6 +140,36 @@ void deserialize() throws Exception {
         .toBuilder().name("ngnix").version("1.2.3").repository("https://example.com/charts").build()));
   }
 
+  @Test
+  void canEnableCasePresevation() throws IOException {
+    // Given
+    HelmConfig helmConfig = new HelmConfig();
+    helmConfig.setPreserveParameterCase(true);
+
+    File file = File.createTempFile("case-preservation-test", ".tmp");
+    file.deleteOnExit();
```

Comment:
Even When I comment out these file creation lines, tests seem to pass. Could this possibly be a leftover?

Reply:
Sure is, I've removed it.